ansible/roles/baselayout/vars/main.yml

```diff
@@ -143,7 +143,8 @@ packages: {
     'gcc-6,g++-6,gcc-8,g++-8,python3.8',
   ],  
 
+  # Default gcc/g++ package is 11.
   ubuntu2204: [
-    'gcc,g++,python2,python3,python-is-python3',
+    'gcc,g++,python3,python-is-python3',
```

Comment:
We shouldn't need python2 anymore, right?

Reply:
Correct. In theory just Python 3 should work. I'm not even sure we need `python-is-python3`.